internal/provider/user_resource_test.go

```diff
@@ -0,0 +1,169 @@
+package provider
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	cloudservicev1 "github.com/temporalio/terraform-provider-temporalcloud/proto/go/temporal/api/cloud/cloudservice/v1"
+)
+
+const (
+	// TODO change email address.
+	emailDomain   = "temporal.io"
+	emailBaseAddr = "test"
```

Comment:
can this be `saas-cicd-prod+terrformprovider-<a random string>`? we've got some automation internally to know not to worry about these :P

Reply:
Changed rn!